qa/tasks/mgr/dashboard/helper.py

```diff
@@ -466,6 +466,18 @@ def find_object_in_list(cls, key, value, iterable):
                 return obj
         return None
 
+    def _wait_for_expected_get_result(self, get_func, get_params, expected_result, max_attempts=30,
```

Comment:
I'm wondering if it makes sense to remove the method completely and to use e.g. https://github.com/ceph/ceph/blob/master/qa/tasks/ceph_test_case.py#L166 instead.

Reply:
yes it does, since it seems `wait_for_expected_get_result` to be broken as it doesn't fail after all tries fail.
I have updated this PR to use  `wait_until_equal` and will create another PR to remove `wait_for_expected_get_result`

Reply:
Great, thank you!